graylog2-radio/src/main/java/org/graylog2/radio/users/NullUserServiceImpl.java

```diff
@@ -103,4 +105,9 @@ public long count() {
     public int delete(String username) {
         return 0;
     }
+
+    @Override
+    public Collection<User> loadAllForRole(Role role) {
+        return null;
```

Comment:
Either return `Collections.emptySet()` (or empty list…) or mark this method as nullable.